app/views/posts/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<h3>This is a list of posts for a given usser</h3>
+
+<% if @user %>
+  <h1>User: <%= @user.name %></h1>
+  <p>Number of Posts: <%= @user.posts_counter %></p>
+
+
+  <h1>All Posts</h1>
+  <ul>
+    <% @posts.each do |post| %>
+      <li>
+        <a href="<%= user_post_path(post.author, post) %>">
+          <%= post.title %>
+        </a>
+        <br>
+        Number of Comments: <%= post.comments_counter %>
+        <br>
+        Number of Likes: <%= post.likes_counter %>
+
+        <ul>
+          <% post.recent_comments.each do |comment| %>
```

Comment:
- Good job done so far, but this method does not exist in your Post model, please fix this and use the correct method 🙏